pkg/subctl/datafile/datafile.go

```diff
@@ -68,7 +68,7 @@ func (data *SubctlData) WriteToFile(filename string) error {
 		return err
 	}
 
-	if err := ioutil.WriteFile(filename, []byte(dataStr), 0644); err != nil {
```

Comment:
Is this going to cause some undesired side-effect not tested in CI?

Reply:
The only difference is that the file won’t be readable by anyone but the user who ran `subctl`, which shouldn’t be an issue in practice.